.github/workflows/ci.yml

```diff
@@ -117,7 +117,7 @@ jobs:
             --no-link-pr \
             --github-token "$GITHUB_TOKEN" \
             ./status/v1-update.json \
-            https://github.com/qchateau/conan-center-bot/issues/4
+            https://github.com/$GITHUB_REPOSITORY/issues/4
```

Comment:
The build of this branch [failed ](https://github.com/ericLemanissier/conan-center-bot/runs/1838586539?check_suite_focus=true)on my repo with
```
update failed: Unauthorized
error while updating: https://github.com/ericLemanissier/conan-center-bot/issues/4
```
Which I don't understand

Reply:
Ah yes this one is tricky. My bad.
I override the default GITHUB_TOKEN env var by another token defined in the action secrets (see the `env` section of both workflows) because the default token does not have the permission to update issues. So I need to use a manually generated token to be able to update issues.